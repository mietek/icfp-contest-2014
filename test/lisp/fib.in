(SETQ FIB-REC (LAMBDA (N)
  (COND ((LESSP N 2) N)
        (T (PLUS (FIB-REC (DIFFERENCE N 1)) (FIB-REC (DIFFERENCE N 2)))))))

(FIB-REC 16)
