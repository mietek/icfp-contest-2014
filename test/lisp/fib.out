ENTERING THE LISP INTERPRETER

{user defined function: APPEND}
{user defined function: REVERSE}
{user defined function: EQUAL}
{builtin function: NOT}
{user defined function: MEMBER}
{user defined function: INTO}
{user defined function: ONTO}
{user defined special form: APPLY}
{user defined function: SORT}
{user defined function: MERGE}
{user defined function: GETPROP}
{user defined function: ASSOC}
{user defined function: PUTPROP}
{user defined function: REMPROP}
{user defined function: NAX}
{user defined function: NX}
*>>{user defined function: FIB-REC}
*>987
*